docs/pages/reference/backends.mdx

```diff
@@ -330,6 +330,13 @@ teleport:
       - postgresql://user_name@database-address/teleport_audit?sslmode=verify-full#disable_cleanup=false&retention_period=2160h
 ```
 
+<Admonition type="note">
```

Comment:
I would put this Admonition higher up in the section so it's more visible. I suggest right after the "The PostgresSQL backend requires..." paragraph, since by that point, this Admonition and the preceding text will have included the same level of detail (i.e., system requirements).